cmd/kube-apiserver/app/options/options.go

```diff
@@ -166,7 +166,7 @@ func (s *ServerRunOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.IntVar(&s.MasterCount, "apiserver-count", s.MasterCount,
 		"The number of apiservers running in the cluster, must be a positive number.")
 
-	fs.StringVar(&s.EndpointReconcilerType, "alpha-endpoint-reconciler-type", string(s.EndpointReconcilerType),
+	fs.StringVar(&s.EndpointReconcilerType, "endpoint-reconciler-type", string(s.EndpointReconcilerType),
```

Comment:
This needs to be deprecated first before being removed. for example https://github.com/kubernetes/kubernetes/pull/51198

Reply:
Edit, forgot that this flag wasn't introduced in 1.8 so no need to deprecate anything.